Automattic/simplenote-android
# New Tracks Event for Checklist Button

Adding a new tracks event for when the checklist button is tapped on the toolbar.

**To Test**
* Ensure analytics is enabled in the app settings.
* Open a note and tap on the checklist button.
* Confirm the event shows up in the tracks live event view. The event is `spandroid_editor_checklist_inserted`
